main/keychain/src/main/java/org/cryptomator/keychain/LinuxSecretServiceKeychainAccessImpl.java

```diff
@@ -57,6 +59,18 @@ public void deletePassphrase(String key) throws KeychainAccessException {
 		}
 	}
 
+	@Override
+	public void changePassphrase(String key, CharSequence passphrase) throws KeychainAccessException {
+		try (SimpleCollection keyring = new SimpleCollection()) {
+			List<String> list = keyring.getItems(createAttributes(key));
+			if (list != null) {
+				keyring.updateItem(list.get(0), LABEL_FOR_SECRET_IN_KEYRING, passphrase, createAttributes(key));
```

Comment:
I guess this keyring doesn't support simply calling `createItem()` again? Then we wouldn't need a completely new method in the `KeychainAccess` interface.

Reply:
Yes, although the low level API offers [an option](https://github.com/swiesend/secret-service/blob/master/src/main/java/org/freedesktop/secret/interfaces/Collection.java#L168) for overwriting a secret (`Item`), the `SimpleCollection` does [not make use](https://github.com/swiesend/secret-service/blob/master/src/main/java/org/freedesktop/secret/simple/SimpleCollection.java#L267) of it.

Instead, `SimpleCollection` offers a method [for updating](https://github.com/swiesend/secret-service/blob/master/src/main/java/org/freedesktop/secret/simple/SimpleCollection.java#L312) an `Item`.

Reply:
Fair enough, might be good to have a specialized method anyway. Maybe some keychain implementations store a history. 👍